src/screens/QuizScreenNoAcctScreen.tsx

```diff
@@ -0,0 +1,337 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native';
+import Icon from 'react-native-vector-icons/Entypo';
+import { RFValue } from 'react-native-responsive-fontsize';
+import CircularTimer from 'react-native-circular-timer';
+import { QuestionResult } from './QuizResultsScreenNoAcctScreen';
+import { OfflineContact } from './noAcctHelperFunctions';
+
+interface QuestionOption {
+  contact: OfflineContact;
+  isCorrect: boolean;
+}
+
+interface Props {
+  navigation: any;
+  route: {
+    params: {
+      contacts: OfflineContact[];
+      questionResults: QuestionResult[];
+      QuizTitleListName: string;
+      CurrentQuizQuestionNumber: number;
+    };
+  };
+}
+
+// interface State {
+//     menuVisible: boolean;
+//     searchQuery: string;
+//     contacts: OfflineContact[];
+//     questionResults: QuestionResult[];
+//     QuizTitleListName: string;
+//     CurrentQuizQuestionNumber: number;
+//     loading: boolean;
+//     refreshing: boolean;
+//   }
+
+class QuizScreenNoAcctScreen extends React.Component<Props> {
+  private timer: CircularTimer | null;
+
+  constructor(props: Props) {
+    super(props);
+    // this.state = {
+    //     menuVisible: false,
+    //     searchQuery: '',
+    //     //contacts: [],
+    //     questionResults: [],
+    //     QuizTitleListName: '',
+    //     loading: true,
+    //     refreshing: false,
+    //   };
```

Comment:
Do we need this?

Reply:
No, we don't. I can remove it sometime today after my classes.